bftengine/src/bcstatetransfer/BCStateTran.cpp

```diff
@@ -1345,6 +1345,11 @@ inline bool BCStateTran::BlocksBatchDesc::isValid() const {
 // State
 //////////////////////////////////////////////////////////////////////////////
 
+void BCStateTran::enableFetchingState(void) {
```

Comment:
move to delegator, we still need to be convinced, that even after you move to delegator - this is a must.
changing target state, is not a good idea usually.

Reply:
you left it here, i dont see it addressed

Reply:
I misunderstood last time. I've addressed it now.